docs/specification/2025-03-26/basic/authorization.md

```diff
@@ -4,242 +4,394 @@ type: docs
 weight: 15
 ---
 
-{{< callout type="info" >}} **Protocol Revision**: 2025-03-26 {{< /callout >}}
+{{< callout type="info" >}} **Protocol Revision**: TBD {{< /callout >}}
 
 ## 1. Introduction
 
 ### 1.1 Purpose and Scope
 
-The Model Context Protocol provides authorization capabilities at the transport level,
-enabling MCP clients to make requests to restricted MCP servers on behalf of resource
-owners. This specification defines the authorization flow for HTTP-based transports.
-
-### 1.2 Protocol Requirements
+The Model Context Protocol provides authorization capabilities at the transport level, enabling MCP
+clients to make requests to restricted MCP servers on behalf of resource owners. This specification
+defines the authorization flow for HTTP-based transports.
+
+The scope of this document is **user authorization** - how a client can authenticate a user to access a
+server and the server determine whether to authorize the user to access its tools, resources, or any
+other capabilities.
+
+This document **does not discuss server-to-server authorization patterns**.
+
+### 1.2 Baseline tenets
+
+1. **MCP server developers are not security experts**. The current specification assumes that developers
+   will be building their own authorization server. That is not something they ever need to do in the
+   context of MCP servers. This also introduces room for serious mistakes that can lead to compromise.
+1. **MCP clients are identity provider (IdP)-agnostic**. We do not and will not expect clients to be
+   "enlightened" in any capacity to support a protected MCP server. This will fragment the ecosystem.
+   The specification explicitly assumes that clients are not enabled in any unique way to support
+   specific IdPs.
+    - **Note:** It is acceptable for clients to be enhanced for specific IdPs, primarily through
+      pre-registration.
+1. **MCP clients perform user authentication without the server.** The MCP server can guide the client
```

Comment:
Reading the description, I think this is not so much about what the Client does, but about what the Server does not do. Perhaps rephrase the heading as 

**User Authentication is not a MCP Server concern**

Reply:
There are also many references still to an IdP which should be replaced with Authorization Server in the spec to help make this clear.   Authorization Servers may federate with an IdP for user authentication and identity.  More concretely, the `issuer` in this spec doesn't need to speak OIDC or be the user's IdP.